main.go

```diff
@@ -14,9 +14,11 @@ import (
 
 var layer int
 var output string
+var latestOnly bool
 
 func init() {
 	flag.StringVar(&output, "output", "", "output file")
+	flag.BoolVar(&latestOnly, "latest-only", false, "download only the latest layer")
```

Comment:
Can you make the flag name `latest`, I have an aversion to hyphenated flag names...